overload.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"encoding/xml"
+	"os"
+)
+
+type xmlOverloads struct {
+	List []xmlOverload `xml:"overload"`
+}
+
+type xmlOverload struct {
+	Name         string `xml:"name,attr"`
+	OverloadName string `xml:"overloadName,attr"`
+
+	ParameterChanges []xmlParameterChange `xml:"parameterChanges>change"`
+}
+
+type xmlParameterChange struct {
+	// Index is the zero-based index of the parameter list.
+	Index int `xml:"index,attr"`
+	// Name describes a change of the parameter name.
+	Name *xmlNameChange `xml:"name"`
+	// Type describes a change of the parameter type.
+	Type *xmlTypeChange `xml:"type"`
+}
+
+type xmlNameChange struct {
+	Value string `xml:"value,attr"`
+}
+
+type xmlTypeChange struct {
+	Name         string `xml:"name,attr"`
+	PointerLevel int    `xml:"pointerLevel,attr"`
```

Comment:
thought: Rather than privileging pointers I wonder if we should consider parsing C parameters including "*" to determine the pointer level?

Reply:
OK, I'll have a look. I'm still trying to wrap my head around the magic within `parseSignature()`, where this is happening for the current code.

Reply:
Turned out, I could reuse the existing function. It does a little more than required, but dissecting it should be a separate action.